accurest-gradle-plugin/src/test/groovy/io/codearte/accurest/plugin/AccurestIntegrationSpec.groovy

```diff
@@ -1,27 +1,57 @@
 package io.codearte.accurest.plugin
 
-import nebula.test.IntegrationSpec
-import nebula.test.functional.GradleRunner
+import org.apache.commons.io.FileUtils
+import org.gradle.testkit.runner.BuildResult
+import org.gradle.testkit.runner.BuildTask
+import org.gradle.testkit.runner.GradleRunner
+import org.gradle.testkit.runner.TaskOutcome
+import org.junit.Rule
+import org.junit.rules.TemporaryFolder
+import spock.lang.Specification
 
 import java.nio.file.Files
 import java.nio.file.Path
 
 import static java.nio.charset.StandardCharsets.UTF_8
 
 /**
- * @author Olga Maciaszek-Sharma 
```

Comment:
Ops, I had a new file and then renamed it to AccurestIntegrationSpec, will fix it